poppy/fresnel.py

```diff
@@ -129,13 +129,16 @@ def __str__(self):
 
 
 class FresnelWavefront(Wavefront):
+    angular_coordinates = False
+    """Should coordinates be expressed in arcseconds instead of meters at the current plane? """
```

Comment:
Yeah, they're tricky, but I think this is a case where we should be using them.

1) It prevents the issue where pixelscale assignment in `Wavefront.__init__` tries to invoke the `FresnelWavefront.pixelscale` property setter and barfs because `angular_coordinates` is not yet set. This lets us move the super() call to the beginning, making the code easier to reason about (don't have to worry about whether things need to be assigned before or after calling the superclass initializer)
2) Sphinx (or one of the astropy extensions) can read docstrings for class attributes but not attributes defined in `__init__`.. This is perhaps minor
3) Someone would have to go out of their way to assign to `FresnelWavefront.angular_coordinates`; assigning to `wavefront_instance.angular_coordinates` will still do the right thing.